google/cloud/apikeys/quickstart/BUILD.bazel

```diff
@@ -20,6 +20,6 @@ cc_binary(
         "quickstart.cc",
     ],
     deps = [
-        "@google_cloud_cpp//:experimental-apikeys",
+        "@google_cloud_cpp//:apikeys",
```

Comment:
I have a similar PR in the works, but also updated `google/cloud/apikeys/quickstart/CMakeLists.txt`.

Reply:
Let's close this if you have a better PR.